target/linux/rockchip/image/rk3399.mk

```diff
@@ -0,0 +1,15 @@
+# 
+# Copyright (C) 2020 Tobias Maedel
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+define Device/pine64_rockpro64
+	DEVICE_VENDOR := Pine64
```

Comment:
Indent for device definitions is two spaces by convention.

```
define Device/pine64_rockpro64
  DEVICE_VENDOR := Pine64
  ...
```

Reply:
Yep, should now be fixed.